src/features/cosoul/api/mutations.ts

```diff
@@ -0,0 +1,23 @@
+import { ValueTypes } from '../../../lib/gql/__generated__/zeus';
+import { client } from '../../../lib/gql/client';
+
+export async function mintCosoulTx(input: ValueTypes['cosouls_insert_input']) {
+  client.mutate(
+    {
+      insert_cosouls_one: [{ object: { ...input } }, { __typename: true }],
+    },
+    { operationName: 'createCosoul' }
+  );
+}
+
+export async function deleteCosoul(tokenId: number) {
+  client.mutate(
+    {
+      delete_cosouls: [
+        { where: { token_id: { _eq: tokenId } } },
```

Comment:
this doesn't work currently, since the tokenId is not populated... but once it's popualted then this should work.